lib/livebook/application.ex

```diff
@@ -169,6 +172,19 @@ defmodule Livebook.Application do
     end
   end
 
+  defp load_lb_env_vars do
+    for {var, value} <- System.get_env(), String.starts_with?(var, "LB_") do
+      "LB_" <> name = var
+
+      Livebook.Secrets.TemporaryStorage.set_secret(%Livebook.Secrets.Secret{
```

Comment:
Perhaps it would be a good use case for persistent term, since we never modify it?